dv/uvm/data/xcelium/xcelium.hjson

```diff
@@ -7,49 +7,83 @@
 
   // Indicate the tool specific helper sources - these are copied over to the
   // {tool_srcs_dir} before running the simulation.
-  tool_srcs:  ["{proj_root}/dv/uvm/data/xcelium/*"]
+  tool_srcs:  ["{proj_root}/hw/dv/tools/xcelium/*"]
```

Comment:
This looks like it might not be quite right?

Reply:
whoops you're right, good catch